swarm/storage/chunker_test.go

```diff
@@ -263,7 +174,7 @@ func TestSha3ForCorrectness(t *testing.T) {
 
 }
 
-func TestDataAppend(t *testing.T) {
```

Comment:
@gbalint why is this commented out?

Reply:
@nonsense accident, I reenabled it in d8ee6a163b535dfaec483f1d04d9134056b5eee1